### ISSUE TITLE
Ch 5 - Add paragraph about math which makes xpub and xprv work

### ISSUE THREAD

**Issue by danra** [bitcoinbook/bitcoinbook#649]
I didn't find an explanation in the text to why the public key extension process shown in Figure 5-11 derives the same branch of public keys which is derived in Figure 5-10.

This isn't complicated, but isn't trivial either. I think adding a paragraph showing why the math makes this work would help deeper understanding.

Such an explanation would rely on the generation formula in page 63, and on the basic properties of multiplication and addition in an elliptic curve. (at least, that's my understanding of how it works - hopefully a correct one).

I can try to write and add the paragraph if there is interest.

**Comment by wbnns** [comment]
@danra 

Yes, please feel free to open a pull request if you'd like to pursue this change. 👍
